tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -184,3 +184,122 @@ def get_resource_params(self, m, r):
         # Docs are wrong :-(
         # https://cloud.google.com/compute/docs/reference/rest/v1/snapshots/delete
         return {'project': project, 'snapshot': snapshot_id}
+
+
+@resources.register('security-policy')
```

Comment:
the actual usage is a little to specific to claim this generic of a name, this appears to be specific to load balancer backend integration.

Reply:
done